streams-activitypub-graph/pom.xml

```diff
@@ -40,6 +40,11 @@
             <artifactId>streams-activitypub-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.streams.activitypub</groupId>
+            <artifactId>streams-activitypub-server</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
server is now a dependency of graph, b/c certain graph implementation classes need to know the base url of the server